addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/handler/RFXComBridgeHandler.java

```diff
@@ -300,10 +298,9 @@ public void packetReceived(byte[] packet) {
                             + "issue at the relevant tracker. Received message: {}", message);
                 }
             } catch (RFXComMessageNotImplementedException e) {
-                logger.debug("Message not supported, data: {}", DatatypeConverter.printHexBinary(packet));
+                logger.debug("Message not supported, data: {}", HexUtils.bytesToHex(packet));
```

Comment:
The `HexUtils.bytesToHex()` method is applied every time. Should we encapsulate it by an `if (logger.isDebugEnabled())`?

Reply:
This one is an exceptional situation so less critical and the only line in the catch, I think the added / gained value her is minimal